src/Microsoft.Diagnostics.EventFlow.Inputs.Etw/StandardTraceEventSession.cs

```diff
@@ -20,15 +20,46 @@ internal class StandardTraceEventSession : ITraceEventSession
         private bool isProcessing;
         private IHealthReporter healthReporter;
 
-        public StandardTraceEventSession(IHealthReporter healthReporter)
+        public StandardTraceEventSession(string sessionNamePrefix, bool cleanupOldSessions, bool restartExisting, IHealthReporter healthReporter)
         {
             Requires.NotNull(healthReporter, nameof(healthReporter));
+            Requires.NotNullOrWhiteSpace(sessionNamePrefix, nameof(sessionNamePrefix));
 
             this.healthReporter = healthReporter;
-            this.inner = new TraceEventSession($"EventFlow-{nameof(EtwInput)}-{Guid.NewGuid().ToString()}", TraceEventSessionOptions.Create);
+
+            string sessionName = null;
+            if (cleanupOldSessions)
+            {
+                sessionName = CleanupMatchingSessions(sessionNamePrefix, restartExisting);
+            }
+            if (sessionName == null)
+                sessionName = $"{sessionNamePrefix }-{Guid.NewGuid().ToString()}";
```

Comment:
(nitpick)

In EventFlow codebase we use curly braces for `if` statement body even if there is only one statement in the body, see lines 31-34 for example. For the sake of consistency please follow this convention--thanks!

(applies to `CleanupMatchingSessions` method too)